src/views/Help/TechnicalReviewBoard/PrepareTrbConsultMeeting.tsx

```diff
@@ -0,0 +1,106 @@
+  const { t } = useTranslation('prepareTrbConsultMeeting');
+  return (
+    <>
+      <MainContent className="grid-container">
+        <HelpBreadcrumb type="Close tab" />
+        <HelpPageIntro
+          type="Technical Review Board"
+          heading={t('title')}
+          subheading={t('description')}
+        />
+
+        <h2 className="font-heading-xl line-height-heading-2 margin-top-3 margin-bottom-2">
+          {t('what.title')}
+        </h2>
+        <div className="line-height-body-5">{t('what.description')}</div>
+
+        <h3 className="margin-top-3 margin-bottom-1">
+          {t('whatToExpect.title')}
+        </h3>
+        <div className="line-height-body-5">
+          {t('whatToExpect.description')}
+        </div>
+
+        <div className="bg-base-lightest margin-top-2 padding-2">
+          <h3 className="margin-top-0 margin-bottom-1">{t('tips.title')}</h3>
+          <ul className="easi-list line-height-body-5">
+            <li>{t('tips.list.0')}</li>
+            <li>{t('tips.list.1')}</li>
+            <li>{t('tips.list.2')}</li>
+            <li>{t('tips.list.3')}</li>
+            <li>{t('tips.list.4')}</li>
+            <li>{t('tips.list.5')}</li>
+          </ul>
+        </div>
+
+        <h2 className="margin-top-6 margin-bottom-2">
+          {t('whatToBring.title')}
+        </h2>
+        <div className="line-height-body-5 margin-bottom-1">
+          {t('whatToBring.description')}
+        </div>
+        <ul className="easi-list line-height-body-5">
+          <li>{t('whatToBring.list.0')}</li>
+          <li>{t('whatToBring.list.1')}</li>
+          <li>{t('whatToBring.list.2')}</li>
+        </ul>
+
+        <SummaryBox
+          heading={t('downloadTemplates.title')}
+          className="desktop:grid-col-6 margin-top-4"
+        >
+          <p>
+            <Trans
+              i18nKey="prepareTrbConsultMeeting:downloadTemplates.description"
+              components={{
+                a: <Link href={`mailto:${CMS_TRB_EMAIL}`}> </Link>,
+                email: CMS_TRB_EMAIL
+              }}
+            />
+          </p>
+          <div>
+            <Link href={TRB_DECK_TEMPLATES} variant="external">
+              {t('downloadTemplates.link')}
+            </Link>
+          </div>
+        </SummaryBox>
+
+        <h2 className="margin-top-6 margin-bottom-2">{t('outcomes.title')}</h2>
+        <div className="line-height-body-5 margin-bottom-1">
+          {t('outcomes.description')}
+        </div>
+        <ul className="easi-list line-height-body-5">
+          <li>{t('outcomes.list.0')}</li>
+          <li>{t('outcomes.list.1')}</li>
+          <li>{t('outcomes.list.2')}</li>
+          <li>{t('outcomes.list.3')}</li>
+        </ul>
+
+        <NeedHelpBox
+          className="desktop:grid-col-6 margin-top-6"
+          content={t('help.text')}
+          email={CMS_TRB_EMAIL}
+        />
+      </MainContent>
+
+      <div className="margin-top-7">
```

Comment:
This margin could be added to the `<MainContent>` component as `margin-bottom-7` instead of having an extra div here.